### ISSUE TITLE
Box86 for Wine

### ISSUE THREAD

**Issue by artdeell** [ptitSeb/box86#143]
Hello!
As you may know, there is a project called Hangover, which is an arm64 Wine and Qemu user space emulator running inside Wine itself. Can Box86 be modified (or ported) to be used like that?

I'm asking because current Hangover can only run Notepad++ now, and can't run anything else. I think the performance and compartiblity will increase if Box86 will replace Qemu user space emulator.

**Comment by ptitSeb** [comment]
This is a vast subject.

Initialy, before even working on box86, I wanted to work on somthing similar to Hangover. But with the complexity of the wine source code, and because I didn't know it well enough, I switched to box86 project, but with the same ideas.

Now, box86 matured, and while I still think it would be better if I start my initial project, I've started working on wine support on box86

Now, injecting box86 dynarec inside hangover is not an option for now. Box86 is 32bits only, while hangover target both x86 and x86_64.

Hopefully, I'll get regular x86 wine running on box86...
